inception-concept-linking/src/main/resources/META-INF/asciidoc/user-guide/projects_concept-linking.adoc

```diff
@@ -0,0 +1,38 @@
+[[sect_projects_concept-linking]]
+=== Concept Linking
+The user can setup a local or remote KB for his project in the settings, which includes the
+configuration for the various IRIs. The necessary IRIs are as follows.
+
+* Class IRI: Declares a resource as a class.
+* Subclass IRI: Allows the declaration of hierarchies of classes.
+* Type IRI: Indicates that resource A is an instance of resource B.
+* Description IRI: Indicates that resource A is a description of resource B.
+
+For Concept Linking, the only relevant IRI is the Description IRI, which needs to be selected to enable Concept Linking functionality.
+Furthermore, the KB must support full-text-search. It is planned to enable full-text-search for local KBs in the next release.
+Checking “Supports Concept Linking” will activate the disambiguation pipeline.
+If it is not checked, Concept Linking still works, but all concepts from the KB are displayed
+in alphabetical order.
+
+A Knowledge Base configured for Concept Linking can be seen in the screenshot below.
+
+image::concept-linking1.png[align="center"]
+
+Documentation about the NamedEntityLinker can be found in the <<sect_projects_recommendation, Project Settings>> .
+
+
+==== Resources
+There are several resources that can be put into the `.inception/resources` folder. These include:
```

Comment:
In order to improve the quality of suggestions, several additional resources can be incorporated. These are to be put into the